services/esSearcherPreprocessorSvc.js

```diff
@@ -43,6 +43,13 @@ angular.module('o19s.splainer-search')
             var hl = { fields: {} };
 
             angular.forEach(fields, function(fieldName) {
+              /*
+               * ES doesn't like highlighting on _id if the query has been filtered on _id using a terms query.
```

Comment:
Should we open a ticket with ES to get this fixed?   Or is this a "works as designed" limitation?

Reply:
It smells like a bug to me tho highlighting on the `_id` field is a bit odd and probably not supported under the hood anyways.  In any case, it would be better if it didn't 500 the request, or at least give a more useful message when it does.